druid/src/main/java/org/apache/calcite/adapter/druid/DruidQuery.java

```diff
@@ -634,9 +634,19 @@ protected QuerySpec getQuery(RelDataType rowType, RexNode filter, List<RexNode>
         ImmutableList.Builder<JsonCollation> colBuilder =
             ImmutableList.builder();
         for (Pair<Integer, Direction> p : Pair.zip(collationIndexes, collationDirections)) {
+          boolean isNumericSort = false;
```

Comment:
i thought about that, but looked a bigger more intrusive change than this one, but i can do it that way.